### ISSUE TITLE
Options question

### ISSUE THREAD

**Issue by piusyikyu** [ultrafunkamsterdam/undetected-chromedriver#656]
I am developing a Kotlin version of your library and I got a question about your options.py.

I have highlighted some lines below. Do you prioritise the preferences saved in the filesystem instead of the options provided in code?

```
def handle_prefs(self, user_data_dir):
        prefs = self.experimental_options.get("prefs")
        if prefs:
            ...
            # undot prefs dict keys
            undot_prefs = {} <-------------------
            for key, value in prefs.items():
                undot_prefs.update(self._undot_key(key, value)) <-------------------

            prefs_file = os.path.join(default_path, "Preferences")
            if os.path.exists(prefs_file):
                with open(prefs_file, encoding="latin1", mode="r") as f:
                    undot_prefs.update(json.load(f))  <-------------------

            with open(prefs_file, encoding="latin1", mode="w") as f:
                json.dump(undot_prefs, f)

           ...
```

**Comment by sebdelsol** [comment]
You're right the default prefs might overwrite those defined by the user. This was a first draft...
And btw this naive merge won't work with nested dictionaries and there some other issues.
Please check this pull request that haven't been merged yet : #558

**Comment by piusyikyu** [comment]
Thanks
